languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -34572,15 +34518,121 @@ Andere Projekte</example>
                 </pattern>
                 <message>Möchten Sie die vom Duden empfohlene Schreibweise <suggestion><match no="1" regexp_match="phantas(.*)" regexp_replace="fantas$1"/></suggestion> verwenden?</message>
                 <example correction="fantastische" type="incorrect">Es ist eine <marker>phantastische</marker> Landschaft.</example>
-		<example correction="fantasievolle" type="incorrect">Er lieferte eine <marker>phantasievolle</marker> Beschreibung.</example>
+                <example correction="fantasievolle" type="incorrect">Er lieferte eine <marker>phantasievolle</marker> Beschreibung.</example>
             </rule>
             <rule>
+                <antipattern>
+                    <token regexp="yes">.*pho[nt]ograph.*|graphe(misch|matisch|ologisch).*</token>
+                </antipattern>
                 <pattern>
-                    <token regexp="yes">geograph.*</token>
+                    <token regexp="yes">.*graph.*</token>
```

Comment:
I've commented this rule out for now, it complained about "Graph" which is the recommended spelling according to http://www.duden.de/rechtschreibung/Graph_Darstellung_Mathematik. Could you also check the other new matches at https://languagetool.org/regression-tests/20170717/result_de_20170717.html?